content/docs/reference/design-docs/stdcm/pathfinding_module/conflict_avoidance/index.en.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Conflict avoidance"
+linkTitle: "4 - Conflict avoidance"
+weight: 40
+---
+
+While exploring the graph, it is possible to end up in places that would
+cause conflicts. They can be avoided by adding delay.
```

Comment:
Just a quick proposal to avoid repeating "cause conflicts" too many times. Feel free to keep the sentence you prefer though.
```suggestion
While exploring the graph, it is possible to end up in locations that would
generate conflicts. They can be avoided by adding delay.
```